src/cpp/session/modules/NotebookHtmlWidgets.R

```diff
@@ -284,6 +284,9 @@
                                             libraryFolder,
                                             chunkOptions)
 {
+   # let others know we're executing
+   options(rstudio.notebook.executing = TRUE)
```

Comment:
Is there a particular reason this is coupled to HTML widget capture? If not, how would you feel about moving it to a more generic home? cf. how we set/unset the 'width' option: 

https://github.com/rstudio/rstudio/blob/2334aeb79e0306107fe8561ed7ea173ce299d81f/src/cpp/session/modules/rmarkdown/NotebookExec.cpp#L222
...
https://github.com/rstudio/rstudio/blob/2334aeb79e0306107fe8561ed7ea173ce299d81f/src/cpp/session/modules/rmarkdown/NotebookExec.cpp#L450

Reply:
That seems like a more natural place -- I'll make that change.